### PR TITLE
Image entity has correct orientation and is collisionless

### PR DESCRIPTION
**Pull request by elisa-lj11**
## Test Plan
1. Drag and drop an image into interface
2. The image should be facing you, and you should be able to walk through it
3. Find an image in the asset browser (.jpg or .png) and click "Add To World"
4. That image should also be facing you, and you should be able to walk through it
5. Go into Create mode, and add an image
6. Switch the Image URL to this: https://i.ytimg.com/vi/4PDQ1gziLL8/maxresdefault.jpg
7. The image should now have a great picture of a doge'n pupper
8. Walk through the doge, dont be skared if he does a lick

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7527/app_12502_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7527/HighFidelity-Beta-PR12502-b56c78d36c96f5c3196915843e8d6b02222a47c9.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7527/HighFidelity-Beta-PR12502-b56c78d36c96f5c3196915843e8d6b02222a47c9.exe

**Comment by elisa-lj11** [comment]
Waiting for https://github.com/highfidelity/hifi/pull/12504/ to be merged before proceeding

**Comment by elisa-lj11** [comment]
PR #12504 was closed, so testing/review may proceed

File: libraries/entities/src/EntityScriptingInterface.cpp [hunk 1]
```diff
@@ -305,7 +305,7 @@ QUuid EntityScriptingInterface::addEntity(const EntityItemProperties& properties
 }
 
 QUuid EntityScriptingInterface::addModelEntity(const QString& name, const QString& modelUrl, const QString& textures, const QString& shapeType,
-                                               bool dynamic, const glm::vec3& position, const glm::vec3& gravity) {
+                                               bool dynamic, bool collisionless, const glm::vec3& position, const glm::vec3& gravity) {
```
Comment (dback2) [inline libraries/entities/src/EntityScriptingInterface.cpp]: Line lengths of this and line above could use adjusting to be <= 128. And maybe 1st header line too while we are fixing it?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7580/app_12502_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7580/HighFidelity-Beta-PR12502-17ddf210281b4e7267179c29167ef5cec8e0a810.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7580/HighFidelity-Beta-PR12502-17ddf210281b4e7267179c29167ef5cec8e0a810.exe

**Comment by misslivirose** [comment]
Does changing Entities.addModelEntity to have a new parameter impact any other places that it's called in the codebase? Does that parameter need a default value?

**Comment by elisa-lj11** [comment]
@misslivirose when I checked the hifi codebase I didn't see any other uses of `Entities.addModelEntity`, unsure about its external use but when I searched the hifi-content repo I couldn't find any instances of it being called

**Comment by qlimnavarro** [comment]
Currently Testing.

**Comment by qlimnavarro** [comment]
Fail in Step 4, the image added to world via asset browser does not face user. All other steps pass successfully. Video and Log attached:
[PR12502 Files.zip](https://github.com/highfidelity/hifi/files/1767638/PR12502.Files.zip)

@conklin94122

**Comment by misslivirose** [comment]
@elisa-lj11 - great. Let's make sure the documentation is up to date for the Entities API since addModelEntity is in there - can you submit a PR to the grav repo?

**Comment by elisa-lj11** [comment]
@misslivirose I don't think `Entities.addModelEntity` is meant to be used by scripters, so I'm not sure if it should be added to the Entities API. In both `TabletAssetServer.qml` and `AssetServer.qml` there's this comment above the (only) calls to `addModelEntity`
```Entities.addEntity doesn't work from QML, so we use this.```

**Comment by elisa-lj11** [comment]
@qlimnavarro I tested with another model (.fbx) and the issue of models not facing you when you rez them is present with models outside of the image entity, so this issue should be resolved outside the scope of this PR.

**Comment by Qliemillar** [comment]
Currently being tested

**Comment by Qliemillar** [comment]
Testing complete, aside from the entity facing away on step 4 (which is acknowledged as being outside the scope of this PR), all steps pass! @conklin94122

**Pull request merged**